src/shared/user-record-show.c

```diff
@@ -206,6 +212,31 @@ void user_record_show(UserRecord *hr, bool show_full_group_info) {
         if (hd)
                 printf("   Directory: %s\n", hd);
 
+        if (hr->blob_directory) {
+                const char *filename;
+                const uint8_t *hash_bytes;
+                unsigned i = 0, n;
```

Comment:
change do size_t. This ultimately counts objects in memory, and that's something we pretty much count in size_t only these days. (and yes, we should fix the hashmap_size() return value accordingly)